aries-site/src/data/wcag/box.json

```diff
@@ -0,0 +1,512 @@
+      "label": "Audio Description or Media Alternative (Prerecorded)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.2.4",
+      "label": "Captions (Live)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.2.5",
+      "label": "Audio Description (Prerecorded)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.2.6",
+      "label": "Sign Language (Prerecorded)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.2.7",
+      "label": "Extended Audio Description (Prerecorded)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.2.8",
+      "label": "Media Alternative (Prerecorded)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.2.9",
+      "label": "Audio-only (Live)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.3.1",
+      "label": "Info and Relationships",
+      "status": "conditional",
+      "notes": "Use semantic HTML to structure content inside the Box (e.g., headings, forms, groups)."
+    },
+    {
+      "rule": "1.3.2",
+      "label": "Meaningful Sequence",
+      "status": "conditional",
+      "notes": "Ensure content inside the Box follows a logical order in both visual layout and HTML structure."
+    },
+    {
+      "rule": "1.3.3",
+      "label": "Sensory Characteristics",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.3.4",
+      "label": "Orientation",
+      "status": "passed",
+      "notes": ""
+    },
+    {
+      "rule": "1.3.5",
+      "label": "Identify Input Purpose",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.3.6",
+      "label": "Identify Purpose",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.4.1",
+      "label": "Use of Color",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.4.2",
+      "label": "Audio Control",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.4.3",
+      "label": "Contrast (Minimum)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.4.4",
+      "label": "Resize Text",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.4.5",
+      "label": "Images of Text",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.4.6",
+      "label": "Contrast (Enhanced)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.4.7",
+      "label": "Low or No Background Audio",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.4.8",
+      "label": "Visual Presentation",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.4.11",
```

Comment:
looks like you have this rule twice

Reply:
Nice catch @britt6612. Once this is fixed, good to merge. Everything else looked good